HW_1/task_3.py

```diff
@@ -0,0 +1,26 @@
+
+# Задача 6: Вы пользуетесь общественным транспортом?
+# Вероятно, вы расплачивались за проезд и получали
+# билет с номером. Счастливым билетом называют такой
+# билет с шестизначным номером, где сумма первых
+# трех цифр равна сумме последних трех. Т.е. билет
+# с номером 385916 – счастливый, т.к. 3+8+5=9+1+6.
+# Вам требуется написать программу, которая проверяет
+# счастливость билета.
+
+# 385916 -> yes
+# 123456 -> no
+
+num = (input('Введите 6-ое число: ')) 
+part1 = int(num[0]) + int(num[1]) + int(num[2])    #Сложение первых трех
+part2 = int(num[3]) + int(num[4]) + int(num[5])    #Сложение вторых трех
+
+if part1 == part2:
+    print('Сложение первых 3-х чисел = ', int(num[0]) + int(num[1]) + int(num[2]))
+    print('Сложение вторых 3-х чисел = ', int(num[3]) + int(num[4]) + int(num[5]))
```

Comment:
У вас ведь уже есть переменные, в которых посчитано это. Почему бы их не использовать?
```suggestion
    print('Сложение первых 3-х чисел = ', part1)
    print('Сложение вторых 3-х чисел = ', part2)
```